src/cqerl_client.erl

```diff
@@ -923,3 +955,10 @@ stop_during_startup(Reason, State = #client_state{users = Users}) ->
     % connect.
     cqerl_app:mode() =:= pooler andalso timer:sleep(200),
     {stop, normal, State#client_state{socket=undefined}}.
+
+
+next_heartbeat_within(#client_state{last_socket_send = LastSocketSend,
+                                    heartbeat_interval = Interval}) ->
+    CurrentSecs = os:system_time(millisecond),
```

Comment:
The variable name is misleading - it's current milliseconds.

Reply:
Fixed, thanks.